### DISCUSSION TITLE
Unable to reverify University

### DISCUSSION THREAD

**Discussion by pedrogsc** [community/community#51143] [GitHub Education]
I'm trying to renew my teacher application, but when I select the school email address I receive the following prompt:

We require applicants to use a school-issued email address to apply.

My university is the University of Coimbra, Portugal
My email address is *******@icnas.uc.pt

My application was accepted the first time in February.

Please, if you can help me.

Thank you

**Comment by samus-aran** [comment]
Hi @pedrogsc 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Comment by pedrogsc** [comment]
I kept retrying and apparently today I have been acepted once more. Thank you anyway for the attention.
